class-udm-updater.php

```diff
@@ -621,6 +679,7 @@ protected function print_plugin_connector_box() {
 				<button class="button button-primary udmupdater-disconnect"><?php _e('Disconnect', 'udmupdater');?></button>
 			</div>
 			<?php } else { ?>
+			<div class="udmupdater_duplicate_site_warning" style="<?php if (!$duplicate_site) echo 'display: none'; ?>"><?php if ($duplicate_site) echo esc_js($this->udmupdaterl10n['duplicate_site_id']); ?></div>
```

Comment:
@andtown I think this should use `htmlspecialchars()` rather than `esc_js()` ?

Reply:
@DavidAnderson684 Nice catch, yes it should use `esc_html()`, which already has `htmlspecialchars()` included within the function.